messages/bluetooth.proto

```diff
@@ -15,6 +15,14 @@
 syntax = "proto3";
 package shiftcrypto.bitbox02;
 
+message BluetoothEnableRequest {
+  bool enable = 1;
+}
+
+message BluetoothEnableResponse {
+  bool enabled = 1;
+}
```

Comment:
No need for this, can just respond with BluetoothSuccess.

Reply:
done